src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -297,25 +288,16 @@ private BindingConfiguration SafeGetBindingConfig()
 
         private void RegisterLocalServices()
         {
-            this.localServices.Add(typeof(ICredentialStoreService), new Lazy<ILocalService>(() => credentialStoreService));
```

Comment:
Miscellaneous: I removed `this` from this method since I was making changes to the method. However, it does mess the GitHub diff up somewhat :-(